praw/models/reddit/subreddit.py

```diff
@@ -1284,6 +1289,10 @@ def __init__(self, subreddit):
         """
         self.subreddit = subreddit
 
+    def __iter__(self):
+        """Abstract method to return flair templates."""
+        raise NotImplementedError
```

Comment:
Could this be `raise NotImplementedError()`?

Reply:
Ok.